priv/apps/itest/lib/eventer.ex

```diff
@@ -0,0 +1,109 @@
+defmodule Itest.Eventer do
```

Comment:
hm, maybe not a big issue, but `Eventer` used to be the module to push notifications out of Watcher via websockets. It kinda sunk in, maybe a different name can be used here, to avoid confusion and search clashes?

not a big deal if you like this name though :)